go.mod

```diff
@@ -24,7 +24,7 @@ require (
 	github.com/apache/arrow/go/arrow v0.0.0-20200610220642-670890229854
 	github.com/apache/thrift v0.0.0-20181211084444-2b7365c54f82 // indirect
 	github.com/armon/circbuf v0.0.0-20150827004946-bbbad097214e
-	github.com/aws/aws-sdk-go v1.16.19
+	github.com/aws/aws-sdk-go v1.33.8
```

Comment:
I followed this - https://github.com/cockroachdb/cockroach/blob/master/build/README.md#dependencies and pushed a named branch to cockroachdb/vendored. Should the go.mod/go.sum changes not show up here?

Reply:
I believe these changes are correct: these changes show up here, and are consistent with the content in your updated `vendor` ref I assume. When this PR merges it'll bump the submodule pointer to your ref in vendored, and these manifest changes will merge so the manifest remains consistent with what is checked out of the submodule.